pynet/models/spherical/unet.py

```diff
@@ -46,7 +46,7 @@ class SphericalUNet(nn.Module):
     """
     def __init__(self, in_order, in_channels, out_channels, depth=5,
                  start_filts=32, conv_mode="1ring", up_mode="interp",
-                 cachedir=None):
+                 merge_mode="concat", icosahedron=icosahedron, cachedir=None):
         """ Initialize the Spherical UNet.
 
```

Comment:
Please update the docstring with the new parameters.
We want to work on icosahedron only (no other geometry is implemented yet).
For the sake of simplicity, do we really need the new icosahedron parameter (not used in the remaining code)?